infrastructure/conf/nginx-conf-deploy/default

```diff
@@ -120,7 +120,8 @@ server {
         include fastcgi_params;
         fastcgi_param SCRIPT_FILENAME /home/site/wwwroot/api/public/index.php;
     }
-    location ~ "^/api(/(.*))?$" {
+
+    location ^~ /api {
```

Comment:
We shouldn't have any requests to _just_ `/api` right, it should always be `/api/something`? Let's change this to 
```suggestion
    location ^~ /api/ {
```

Reply:
This should apply to both nginx files ;)

Reply:
Fixed in [update rule to require subpath](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11071/commits/05732d9865c5aaa06184404801fb4405cc5aa614)